invenio_app_ils/circulation/utils.py

```diff
@@ -70,6 +71,8 @@ def circulation_default_extension_duration(loan):
 
 def circulation_default_extension_max_count(loan):
     """Return a default extensions max count."""
+    if 'librarian' in current_user.roles or 'admin' in current_user.roles:
```

Comment:
Not quire sure if this is the right place for this.

Reply:
If we do that for circulation let me note that we will have to adjust `_ensure_valid_extension` accordingly.

Reply:
I created an issue https://github.com/inveniosoftware/invenio-circulation/issues/128 for future reference